Automattic/wp-calypso
# TS: Add missing package references

#### Proposed Changes

This PR is adding a few missing package references to fix these TS errors:

![Screenshot 2022-11-22 at 13 16 53](https://user-images.githubusercontent.com/8436925/203300969-f56e6a7b-6586-4885-a317-2b3851b0e924.png)

#### Testing Instructions

* Checkout this branch
* Run `yarn run tsc --project client`
* Verify it completes without errors.

#### Pre-merge Checklist
